### PR TITLE
Merge temp to master branch

### PR DESCRIPTION
**Pull request by Afaqrehman98**
Temporary UI

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -15,4 +16,13 @@
         app:layout_constraintRight_toRightOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
+    <Button
```
Comment (Shashank02051997) [inline app/src/main/res/layout/activity_main.xml:19]: Remove button code

**Review by Shashank02051997 — COMMENTED** [review]
Please do the suggested changes

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (Afaqrehman98) [reply app/src/main/res/layout/activity_main.xml:19]: Done

**Review by Shashank02051997 — APPROVED** [review]
LGTM

**Pull request merged**